ng-dev/utils/git/git-client.ts

```diff
@@ -197,6 +197,11 @@ export class GitClient {
     );
   }
 
+  /** Adds the provided files to the staging area. */
+  stageFiles(files: string[]) {
+    return gitOutputAsArray(this.runGraceful(['add', files.join(' ')]));
```

Comment:
I think it makes more sense to return the command result.  Additionally allowing for the options to be passed to the `runGraceful` method to allow for suppressing output as desired.

Additionally, since the arguments are provided to `spawnSync` as an array, there is not a need to join them, we can just spread the files array into the args array.

```suggestion
  stageFiles(files: string[], options: GitCommandRunOptions = {}): SpawnSyncReturns<string> {
    return this.runGraceful(['add', ...files], options);
```